packages/order-utils/package.json

```diff
@@ -0,0 +1,76 @@
+{
+    "name": "@0xproject/order-utils",
+    "version": "0.0.1",
+    "description": "0x order utils",
+    "main": "lib/src/index.js",
+    "types": "lib/src/index.d.ts",
+    "scripts": {
+        "build:watch": "tsc -w",
+        "build": "tsc && copyfiles -u 3 './lib/src/monorepo_scripts/**/*' ./scripts",
+        "test": "run-s clean build run_mocha",
+        "test:circleci": "yarn test:coverage",
+        "run_mocha": "mocha lib/test/**/*_test.js --bail --exit",
+        "test:coverage": "nyc npm run test --all && yarn coverage:report:lcov",
+        "coverage:report:lcov": "nyc report --reporter=text-lcov > coverage/lcov.info",
+        "clean": "shx rm -rf lib scripts",
+        "lint": "tslint --project .",
+        "manual:postpublish": "yarn build; node ./scripts/postpublish.js",
+        "docs:stage": "yarn build && node ./scripts/stage_docs.js",
+        "docs:json": "typedoc --excludePrivate --excludeExternals --target ES5 --json $JSON_FILE_PATH $PROJECT_FILES",
+        "upload_docs_json": "aws s3 cp generated_docs/index.json $S3_URL --profile 0xproject --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers --content-type application/json"
+    },
+    "config": {
+        "postpublish": {
+            "docPublishConfigs": {
+                "extraFileIncludes": [
+                    "../types/src/index.ts"
+                ],
+                "s3BucketPath": "s3://doc-jsons/order-utils/",
+                "s3StagingBucketPath": "s3://staging-doc-jsons/order-utils/"
+            }
+        }
+    },
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/0xProject/0x-monorepo.git"
+    },
+    "bugs": {
+        "url": "https://github.com/0xProject/0x-monorepo/issues"
+    },
+    "homepage": "https://github.com/0xProject/0x-monorepo/packages/order-utils/README.md",
+    "devDependencies": {
+        "@0xproject/monorepo-scripts": "^0.1.18",
+        "@0xproject/dev-utils": "^0.3.6",
+        "@0xproject/tslint-config": "^0.4.16",
+        "@types/lodash": "4.14.104",
+        "chai": "^4.0.1",
+        "chai-as-promised": "^7.1.0",
+        "chai-bignumber": "^2.0.1",
+        "dirty-chai": "^2.0.1",
+        "sinon": "^4.0.0",
+        "mocha": "^4.0.1",
+        "copyfiles": "^1.2.0",
+        "npm-run-all": "^4.1.2",
+        "typedoc": "0xProject/typedoc",
+        "shx": "^0.2.2",
+        "tslint": "5.8.0",
+        "typescript": "2.7.1"
+    },
+    "dependencies": {
+        "@0xproject/assert": "^0.2.7",
+        "@0xproject/types": "^0.6.1",
+        "@0xproject/json-schemas": "^0.7.21",
+        "@0xproject/typescript-typings": "^0.2.0",
+        "@0xproject/web3-wrapper": "^0.6.1",
+        "@0xproject/utils": "^0.5.2",
+        "@types/node": "^8.0.53",
```

Comment:
Hm... In most packages `@types/node` is a devDep. Why is it a dep here?

Reply:
Copied from json-schemas.